packages/dispatcher-converter/src/converter/amsDispatcherConfigConverter.js

```diff
@@ -57,6 +57,30 @@ class AEMDispatcherConfigConverter {
         this.checkVhosts();
         this.replaceVariableInFarmFiles();
         this.removeNonWhitelistedDirectives();
+        this.createFarmSymLinks(
+            path.join(
```

Comment:
No need to pass the params, since they can be accessed directly inside the methods.